doc/index.rst

```diff
@@ -33,9 +33,12 @@ right next to your ``setup.py`` file::
     # content of: tox.ini , put in same dir as setup.py
     [tox]
     envlist = py27,py36
+
     [testenv]
-    deps=pytest       # install pytest in the venvs
-    commands=pytest  # or 'nosetests' or ...
+    deps = pytest       # install pytest in the virtualenv where commands will be executed
+    commands =
```

Comment:
a matter of style, but do we need here the line break? shouldn't the comment came right after the = and align alongside that all subsequent lines? mostly to keep it consistent with the deps line, also in Python according to PEP-8 two spaces are recommended before the comment, should the deps line follow this rule, to have the most consistent experience?